### ISSUE TITLE
Resets when sync time comes

### ISSUE THREAD

**Issue by kiralikbeyin** [gmag11/NtpClient#3]
20:34:51 03/08/2016
20:34:52 03/08/2016
20:34:54 03/08/2016
20:34:54 03/08/2016
Skipped 53. sec
20:39:52 03/08/2016
20:39:53 03/08/2016
20:39:55 03/08/2016
20:39:55 03/08/2016
re skip 54
20:44:52 03/08/2016
20:44:53 03/08/2016
20:44:54 03/08/2016
20:44:56 03/08/2016
skipped 55 sec

when i don't do anything in browser it is not resetting, if i go on admin.html then to another html and other etc... sync time is making trouble
last reset : 20:49:54 03/08/2016
20:49:55 03/08/2016
Fatal exception 0(IllegalInstructionCause):
i hope you can find a solution to that

And i am using https://github.com/me-no-dev/ESPAsyncWebServer which is not accepting delays or yield

**Comment by kiralikbeyin** [comment]
ok changed cpp code to this and update it to every 1 minute test is ok no reset.

```

    uint32_t beginDelay = millis();
    if (millis() - beginDelay < 10) {
        client->_webClient.flush();
        client->_webClient.stop();
        if (!error)
            return makeTime(time);
        else
            return (0);
    }

//  delay(10);
//  client->_webClient.flush();
//  client->_webClient.stop();
//  if (!error)
//      return makeTime(time);
//  else
//      return (0);
}
#endif // WEB_TIME_SYNC
```

**Comment by gmag11** [comment]
Look at my example here. I've run NTP for hours updating every 5 minutes without problem

**Comment by gmag11** [comment]
Do not use WEB_TIME_SYNC. I will remove it as it is not reliable. NTP is far quicker and better.

**Comment by gmag11** [comment]
Released v2.0.0.
